packages/Testimonial/Testimonial.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Box from '@tds/core-box'
+import Card from '@tds/core-card'
+import Image from '@tds/core-image'
+import Paragraph from '@tds/core-paragraph'
+import safeRest from '@tds/shared-safe-rest'
+
+import styles from './Testimonial.scss'
+
+/**
+ * @version ./package.json
+ */
+/** Testimonial component for displaying testimonial in a standalone, pre-styled component so all testimonials look consistent across the website */
+const Testimonial = ({ testimonial, imgSrc, imgAlt, title, additionalInfo, ...rest }) => {
+  return (
+    <Card {...safeRest(rest)}>
+      <Box between={3}>
+        <Paragraph>&quot;{testimonial}&quot;</Paragraph>
```

Comment:
The use of `&quot;` as a hard-coded value actually isn't `i18n` friendly, different languages use different characters as quotation marks.

**Recommendation:**
By simplifying: ~`<Paragraph>&quot;{testimonial}&quot;</Paragraph>`~
to: `<Paragraph>{testimonial}</Paragraph>`
we can allow the consumer of the component to provide whatever quotation marks are appropriate for their `i18n` use case in the value passed in the  `testimonial` property.

Furthermore, it would be more typographically correct to use `&ldquo;` and `&rdquo;` for English, and (_I think_) `&laquo;` and `&raquo;` in French.

Reply:
fixed